UPGRADE.md

```diff
@@ -8,6 +8,20 @@ awareness about deprecated code.
 
 # Upgrade to 3.0
 
+## BC Break: removed `AnnotationDriver`
+
+It has been deprecated in favor of `ColocatedMappingDriver`.
```

Comment:
s/deprecated/removed everywhere I think :)

Reply:
What I mean is: we removed `AnnotationDriver` because it has been deprecated in favor of `ColocatedMappingDriver`.

Reply:
Ah, I've read that the other way. Anyway it's inconsistent with next paragraphs this way :)

Reply:
I used another phrasing 👍